example_cluster/dind-cluster-v1.11.sh

```diff
@@ -0,0 +1,2049 @@
+#!/bin/bash
```

Comment:
A 2049 line bash script? Sure why not!!! :)

Reply:
:-P at least I'm just vendoring it: https://github.com/kubernetes-sigs/kubeadm-dind-cluster